map/framework.cpp

```diff
@@ -517,10 +517,16 @@ bool Framework::OnCountryFileDelete(storage::TCountryId const & countryId, stora
 
 void Framework::OnMapDeregistered(platform::LocalCountryFile const & localFile)
```

Comment:
А я не в сет прошу вставить документацию, а в этот protected метод фреймворка. Потому что в нем нет ни слова про MwmSet и многопоточность.

> On Mar 25, 2016, at 23:39, ygorshenin <notifications@github.com> wrote:
> 
> In map/framework.cpp <https://github.com/mapsme/omim/pull/2523#discussion_r57468400>:
> 
> > @@ -517,10 +517,16 @@ bool Framework::OnCountryFileDelete(storage::TCountryId const & countryId, stora
> >  
> >  void Framework::OnMapDeregistered(platform::LocalCountryFile const & localFile)
> Специально для тех кому в лом читать документацию к mwm_set.hpp:
> 
>   // An Observer interface to MwmSet. Note that these functions can
>   // be called from *ANY* thread because most signals are sent when
>   // some thread releases its MwmHandle, so overrides must be as fast
>   // as possible and non-blocking when it's possible.
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2523/files/8bff1dbc61d252545ea4b21bffd137f33f43a0e2#r57468400>